packages/pieces/community/google-drive/src/lib/triggers/new-file.ts

```diff
@@ -40,6 +42,12 @@ export const newFile = createTrigger({
   props: {
     parentFolder: common.properties.parentFolder,
     include_team_drives: common.properties.include_team_drives,
+    getFileContent: Property.Checkbox({
```

Comment:
rename `include_file_content`

and this is better suggestion:

```ts
    include_file_content: Property.Checkbox({
      displayName: 'Include File Content',
      description: 'Include the file content in the output. This will increase the time taken to fetch the files and might cause issues with large files.',
      required: false,
      defaultValue: false
    }),
    ```